### PR TITLE
Correct Quarkus Platform Nightly

### PR DESCRIPTION
**Pull request by radtriste**
Drools and OptaPlanner versions are now decorrelated

### CODE REVIEW

File: .ci/jenkins/Jenkinsfile.nightly.quarkus-platform [hunk 1]
```diff
@@ -56,27 +56,34 @@ pipeline {
                     }
                     dir('optaplanner') {
                         deleteDir()
-                        checkout(githubscm.resolveRepository('optaplanner', getGitAuthor(), util.calculateTargetReleaseBranch(getGitBranch(), 7), false))
+                        // Get current corresponding branch and if not working, latest tag
+                        githubscm.checkoutIfExists('optaplanner', getGitAuthor(), util.calculateTargetReleaseBranch(getGitBranch(), 7), getGitAuthor(), githubscm.getPreviousTag(''), true)
```
Comment (mbiarnes) [inline .ci/jenkins/Jenkinsfile.nightly.quarkus-platform]: @radtriste  getGitAuthor() is needed here twice?

**Review by mbiarnes — APPROVED** [review]
LGTM

File: .ci/jenkins/Jenkinsfile.nightly.quarkus-platform (hunk 1 above)
Comment (radtriste) [reply .ci/jenkins/Jenkinsfile.nightly.quarkus-platform]: gonna change this anyway ...

**Comment by radtriste** [comment]
depends on https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/281

**Comment by radtriste** [comment]
Depends on https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/281

File: .ci/jenkins/Jenkinsfile.nightly.quarkus-platform [hunk 2]
```diff
@@ -56,27 +56,42 @@ pipeline {
                     }
                     dir('optaplanner') {
                         deleteDir()
-                        checkout(githubscm.resolveRepository('optaplanner', getGitAuthor(), util.calculateTargetReleaseBranch(getGitBranch(), 7), false))
+                        // Get current corresponding branch and if not working, latest tag
+                        String opBranch = util.calculateTargetReleaseBranch(getGitBranch(), 7)
+                        try {
+                            checkout(githubscm.resolveRepository('optaplanner', getGitAuthor(), opBranch, false))
+                        } catch(err) {
+                            echo "Error checking out ${util.calculateTargetReleaseBranch(getGitBranch(), 7)}. Fallback to latest tag."
```
Comment (lampajr) [inline .ci/jenkins/Jenkinsfile.nightly.quarkus-platform]:
```suggestion
                            echo "Error checking out ${opBranch}. Fallback to latest tag."
```

**Review by lampajr — APPROVED** [review]
LGTM, added just a minor to avoid calling again the method

**Review by rgdoliveira — APPROVED** [review]

**Pull request merged**